WordPress/Classes/ViewRelated/Blog/SharingAccountViewController.swift

```diff
@@ -283,3 +324,13 @@ import WordPressShared
     func didDismissSharingAccountViewController(_ controller: SharingAccountViewController)
     func sharingAccountViewController(_ controller: SharingAccountViewController, selectedKeyringConnection keyringConnection: KeyringConnection, externalID: String?)
 }
+
+
+extension SharingAccountViewController: NoResultsViewControllerDelegate {
+    func actionButtonPressed() {
+        if let url = URL(string: "https://en.support.wordpress.com/publicize/#facebook-pages") {
```

Comment:
@aerych do you think we should show the blog post instead?

Reply:
This should match the behavior in Calypso where the support page is linked from the message shown when attempting to link a Facebook account with no pages.  The blog post is linked from the message shown for a pre-existing connection that needs to be disconnected.

Unless I somehow got that backwards 😬